src/application/core_application.cpp

```
@@ -0,0 +1,54 @@
+  \file   core_application.cpp
+  \brief  Implementations for \ref toy::application::CoreApplication.
+*/
+
+#include "application.hpp"
+
+namespace toy::application {
+
+static CoreApplication * s_instance{nullptr};
+
+CoreApplication::CoreApplication(assertion::AssertionCallback assertionCallback,
+                                 assertion::StackWalkCallback stackWalkCallback) noexcept {
```

Cppcheck (2.20.0)

[style] The function 'version' is never used.

(unusedFunction)